xline-client/src/clients/auth.rs

```diff
@@ -66,6 +66,31 @@ impl AuthClient {
     /// # Errors
     ///
     /// This function will return an error if the inner CURP client encountered a propose failure
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use xline_client::{error::Result, Client, ClientOptions};
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<()> {
+    ///     let curp_members = [
+    ///         ("server0", "10.0.0.1:2379"),
+    ///         ("server1", "10.0.0.2:2379"),
+    ///         ("server2", "10.0.0.3:2379"),
+    ///     ];
+    ///
+    ///     let client = Client::connect(curp_members, ClientOptions::default())
+    ///         .await?
+    ///         .auth_client();
+    ///
+    ///     let _resp = client.auth_enable().await?;
+    ///
+    ///     // auth with some user
+    ///
+    ///     Ok(())
+    /// }
```

Comment:
Add main function to comment example is redundant, FYI: [Rust Std](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.dedup_by_key).
The other comments too.

Reply:
The main function is required here as the code need a runtime to compile and also need to return error.